lib/events.js

```diff
@@ -112,12 +112,13 @@ EventEmitter.init = function(opts) {
   }
 };
 
+const apply = Reflect.apply;
```

Comment:
You can just use `Reflect.apply`, since Reflect here is coming from primordials

Reply:
That's what's giving us the speed bump.